components/admin_console/custom_url_schemes_setting.jsx

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import * as Utils from 'utils/utils';
+
+import Setting from './setting';
+
+export default class TextSetting extends React.Component {
```

Comment:
0/5 but It would be easier to search component whenever the default class name has the same wordings as its filename.

Reply:
Oops. Copy-pasted code strikes again.